owltorrent-core/src/main/java/edu/rice/owltorrent/core/entity/Torrent.java

```diff
@@ -2,8 +2,75 @@
 
 import lombok.Data;
 
+/** Represent the Torrent class after parsing. */
 @Data
 public class Torrent {
   // FIXME: temporary for hello world
-  private final String peerIPAddress;
+  private String peerIPAddress;
```

Comment:
Sorry plz remove this, I had this field only for testing purposes. An actual torrent file shouldn't have this.